### PR TITLE
PUBSTWO-1645 Pull list from SIPP

### PR DESCRIPTION
**Pull request by dsteinich**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,9 @@ PUBS_WAREHOUSE_ENPOINT: http://pubs.er.usgs.gov
 PUBS_AUTHORIZED_GROUPS: group1, group2, spnGroup
 PUBS_SPN_GROUPS: spnGroup
 
+DISSEMINATION_SCHEDULE: <cron type syntac a hyphen for not scheduled>
```
Comment (mbucknell) [inline README.md]: typo 'syntac'

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -22,7 +22,11 @@ server:
   servlet:
     context-path: ${SERVER_CONTEXT_PATH}
 
-#
+sipp:
+  dissemination:
+    schedule: ${DISSEMINATION_SCHEDULE}
+    list-url: ${DISSEMINATION_LIST_URL}
+    daysLastDisseminated: ${DAYS_LAST_DISSEMINATED}
 #ipds:
```
Comment (mbucknell) [inline src/main/resources/application.yml:30]: Should this be removed?

File: src/test/java/gov/usgs/cida/pubs/busservice/ipds/IpdsProcessTest.java [hunk 3]
```diff
@@ -194,7 +191,7 @@ public void okToProcessTest() {
 		newMpPub.setPublicationType(pubType);
 
 		//Good Dissemination (brand new)
-		assertTrue(ipdsProcess.okToProcess(ProcessType.DISSEMINATION, newMpPub, new MpPublication()));
+		assertFalse(ipdsProcess.okToProcess(ProcessType.DISSEMINATION, newMpPub, new MpPublication()));
```
Comment (mbucknell) [inline src/test/java/gov/usgs/cida/pubs/busservice/ipds/IpdsProcessTest.java:194]: I don't understand what this testing? Dissemination is done elsewhere now.

File: src/main/java/gov/usgs/cida/pubs/dao/ipds/IpdsMessageLogDao.java [hunk 4]
```diff
@@ -26,50 +22,40 @@ public IpdsMessageLogDao(SqlSessionFactory sqlSessionFactory) {
...
-	 */
 	@Transactional(readOnly = true)
 	@Override
 	public List<Map<String, Object>> getFromIpds(Integer ipdsMessageLogId) {
 		return getSqlSession().selectList(NS + ".getMpPublicationFromIpds", ipdsMessageLogId);
 	}
 
+	@Transactional(readOnly = true)
+	@Override
+	public List<Map<String, Object>> getFromSipp(Integer ipdsMessageLogId) {
```
Comment (mbucknell) [inline src/main/java/gov/usgs/cida/pubs/dao/ipds/IpdsMessageLogDao.java:57]: Seems a little strange that sipp messages are part of of the IPDSMessageLogDao?

**Review by mbucknell — APPROVED** [review]
A couple of nits and a couple of questions.

**Review by ssoper-usgs — APPROVED** [review]

File: src/test/java/gov/usgs/cida/pubs/busservice/ipds/IpdsProcessTest.java (hunk 3 above)
Comment (dsteinich) [reply src/test/java/gov/usgs/cida/pubs/busservice/ipds/IpdsProcessTest.java:194]: This is just testing that a dissemination will no longer be okToProcess in this manner.

File: src/main/java/gov/usgs/cida/pubs/dao/ipds/IpdsMessageLogDao.java (hunk 4 above)
Comment (dsteinich) [reply src/main/java/gov/usgs/cida/pubs/dao/ipds/IpdsMessageLogDao.java:57]: This will be cleaned up in the next JIRA task.

**Pull request merged**